### PR TITLE
Centering fade formula

### PR DESCRIPTION
**Pull request by fsih**
Change the formula for fading the center crosshair so that the crosshair is always opaque anywhere over the selection

Demo is up at https://fsih.github.io/scratch-gui/centering/

### CODE REVIEW

File: src/helper/selection-tools/bounding-box-tool.js [hunk 1]
```diff
@@ -212,15 +212,20 @@ class BoundingBoxTool {
...
-            const vRoundRect = new paper.Path.Rectangle(vRect, new paper.Size(1, 1));
-            const hRect = new paper.Rectangle(new paper.Point(-6, -1), new paper.Size(12, 2));
-            const hRoundRect = new paper.Path.Rectangle(hRect, new paper.Size(1, 1));
-            anchorIcon = vRoundRect.unite(hRoundRect);
-            vRoundRect.remove();
-            hRoundRect.remove();
+
+            const vRect = new paper.Path.Rectangle({
+                point: [-1, -6],
+                size: [2, 12],
```
Comment (paulkaplan) [inline src/helper/selection-tools/bounding-box-tool.js:218]: Why these changes?

File: src/helper/selection-tools/move-tool.js [hunk 2]
```diff
@@ -153,16 +158,43 @@ class MoveTool {
...
             // Show the center crosshair above the selected item while dragging.
             getDragCrosshairLayer().visible = true;
             this.firstDrag = false;
         }
-        const opacity = Math.max(0,
-            1 - ((CENTER.getDistance(this.selectionCenter.add(dragVector)) / CENTER.x) * (4 * paper.view.zoom)));
-        getDragCrosshairLayer().opacity = opacity;
+
+        let opacityMultiplier = 1;
+        const newCenter = this.selectionCenter.add(dragVector);
```
Comment (paulkaplan) [inline src/helper/selection-tools/move-tool.js:178]: Ooof this really needs some documentation explaining what the intention of this is.

**Review by paulkaplan — DISMISSED** [review]
Code seems to work ok but I really think that opacity multiplier section needs a code comment describing its purpose. LG other than that.

File: src/helper/selection-tools/move-tool.js (hunk 2 above)
Comment (fsih) [reply src/helper/selection-tools/move-tool.js:178]: Added comments

File: src/helper/selection-tools/bounding-box-tool.js (hunk 1 above)
Comment (fsih) [reply src/helper/selection-tools/bounding-box-tool.js:218]: The code from before was leaving behind rectangles in the paper.view, which you could get in your selection by selecting all, and were weird immovable objects
